pkg/api/v1/types.go

```diff
@@ -2002,6 +2002,8 @@ type ListOptions struct {
 	// When specified with a watch call, shows changes that occur after that particular version of a resource.
 	// Defaults to changes from the beginning of history.
 	ResourceVersion string `json:"resourceVersion,omitempty"`
+	// FIXME:
+	TimeoutSeconds *int64
```

Comment:
Wait, don't we already support "timeout"?  So this would be two timeout
flags?

On Oct 16, 2015, at 12:24 PM, Daniel Smith <notifications@github.com> wrote:

In pkg/api/v1/types.go
<https://github.com/kubernetes/kubernetes/pull/15705#discussion_r42261110>:

> @@ -2002,6 +2002,8 @@ type ListOptions struct {
>  	// When specified with a watch call, shows changes that occur after that particular version of a resource.
>  	// Defaults to changes from the beginning of history.
>  	ResourceVersion string `json:"resourceVersion,omitempty"`
> +	// FIXME:
> +	TimeoutSeconds *int64

json:"timeoutSeconds,omitempty"

—
Reply to this email directly or view it on GitHub
<https://github.com/kubernetes/kubernetes/pull/15705/files#r42261110>.